core/jvm/src/test/scala/cats/effect/IOAppSpec.scala

```diff
@@ -35,50 +35,56 @@ class IOAppSpec extends Specification {
     "evaluate and print hello world" in {
       val h = java(HelloWorld, Nil)
       h.awaitStatus() mustEqual 0
-      h.stdout() mustEqual "Hello, World!\n"
+      h.stdout() mustEqual s"Hello, World!${System.lineSeparator()}"
     }
 
     "pass all arguments to child" in {
       val expected = List("the", "quick", "brown", "fox jumped", "over")
       val h = java(Arguments, expected)
       h.awaitStatus() mustEqual 0
-      h.stdout() mustEqual expected.mkString("", "\n", "\n")
+      h.stdout() mustEqual expected.mkString("", System.lineSeparator(), System.lineSeparator())
     }
 
     "run finalizers on TERM" in {
-      import _root_.java.io.{BufferedReader, FileReader}
-
-      // we have to resort to this convoluted approach because Process#destroy kills listeners before killing the process
-      val test = File.createTempFile("cats-effect", "finalizer-test")
-      def readTest(): String = {
-        val reader = new BufferedReader(new FileReader(test))
-        try {
-          reader.readLine()
-        } finally {
-          reader.close()
+      if (System.getProperty("os.name").contains("Windows")) {
```

Comment:
Older versions of windows don't use capitalization. This is rather annoying. The workaround:

```suggestion
      if (System.getProperty("os.name").toLowerCase.contains("windows")) {
```

Reply:
Done.